.vscode/tasks.json

```diff
@@ -0,0 +1,21 @@
+{
+  "version": "2.0.0",
+  "tasks": [
+    {
+      "group": {
```

Comment:
Added this which helps VS Code understand it's actually a `build` task and to treat it as the default.